app/src/main/java/com/mashup/ipdam/ui/profile/ProfileFragment.kt

```diff
@@ -10,6 +12,21 @@ class ProfileFragment : BaseFragment<FragmentProfileBinding>(R.layout.fragment_p
 
     override var logTag: String = "ProfileFragment"
 
+    override fun initLayout() {
+        initVersionName()
+    }
+
+    private fun initVersionName() {
+        try {
+            val packageInfo = requireContext().packageManager.getPackageInfo(
+                requireContext().packageName, 0
+            )
+            binding.profileVersionValue.text = packageInfo.versionName
+        } catch (e : PackageManager.NameNotFoundException) {
+            Log.e(logTag, e.message ?: "PackageManagerName Not Found")
```

Comment:
하드코딩 바꿔줘야하긴 한데 이런식으로 하나 추가해주면 좋을것 같습니당
```suggestion
            Log.e(logTag, e.message ?: "PackageManagerName Not Found")
            binding.profileVersionValue.text = "버전 내용을 찾을 수 없습니다" 
```

Reply:
좋습니다!